docs/_rpc/server.md

```diff
@@ -18,72 +27,197 @@ For example, the `eth_call` method resides in the `eth` namespace.
 Access to RPC methods can be enabled on a per-namespace basis. Find
 documentation for individual namespaces in the sidebar.
 
-### HTTP Server
-
-To enable the HTTP server, use the `--http` flag.
-
-    geth --http
-
-By default, geth accepts connections from the loopback interface (127.0.0.1).
-The default listening port is 8545. You can customize address and port using the
-`--http.port` and `--http.addr` flags.
+## RPC
 
-    geth --http --http.port 3334
+[RPC (Remote Procedure Calls)][rpc] are programs that execute a procedure in an address
```

Comment:
Not sure if I would say RPCs are programs? They are a communication mechanism between programs.